### DISCUSSION TITLE
Undiagnosed Error

### DISCUSSION THREAD

**Discussion by ghost** [tweepy/tweepy#2048] [Questions] [answered]
My twwpy is giving output as Tweeted Slide1.JPG but image is not being published on twitter. 

Where can I find information about what I am doing wrong?

**Comment by Harmon758** [comment]
This isn't actionable at all, as there is no information here.
You need to provide the relevant code and/or full traceback, with the expected and actual behavior, for anyone to be able to help you.

**Comment by ghost** [comment]
import tweepy
import configparser
import os
import threading
import time
import sys

WAIT_TIME = 300000
done = False

def get_tokens():
    api_key = 'key'
    api_secret_key = 'key secret'
    access_token_key = 'token key'
    access_token_secret = 'token secret'
    return api_key, api_secret_key, access_token_key, access_token_secret


def animate():
    for c in range(WAIT_TIME, -1, -1):
        if done:
            break
        sys.stdout.write('\rWaiting for ' + str(c) + ' secs.... ')
        sys.stdout.flush()
        time.sleep(1)
    sys.stdout.write('\rTweeting\n')


def write_tweeted_img(file_name):
    file = open(r"new.txt", 'a') #file path
    file.write('{}\n'.format(file_name))
    file.close()


def check_img_tweeted(file_name):
    with open(r"new.txt") as f: #file path
        if file_name in f.read():
            return True
    return False


if __name__ == '__main__':
    consumer_key, consumer_secret, access_token, access_secret = get_tokens()
    count = 0

    print('Connecting to Twitter...')
    auth = tweepy.OAuthHandler(consumer_key, consumer_secret)
    auth.set_access_token(access_token, access_secret)
    api = tweepy.API(auth)
    print('Connected!\n')

    os.chdir(r"") #imge directory path
    for image in os.listdir(r""): #imge directory path

        if check_img_tweeted(image):
            print('Already Tweeted {} before'.format(image))
            os.remove(image)
            continue

        try:
            api.search_tweets(image)
            print('Tweeted {}'.format(image))
            write_tweeted_img(image)
            count += 1
            os.remove(image)
            done = False

        except TweepError:
            continue

        t = threading.Thread(target=animate)
        t.start()
        time.sleep(WAIT_TIME)
        done = True

    print('\nSuccessfully Tweeted {} Images'.format(count))`

Code being used

**Comment by Harmon758** [comment, marked as answer]
Nowhere in this code is there an image being uploaded to Twitter's API, let alone Tweeted.

I'm not sure what error you're referring to, but you're attempting to handle an exception that no longer exists in the latest versions of Tweepy, which wouldn't be properly imported anyway, and simply suppressing it with no logging/output.

For code block usage, see https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks.
